components/checkbox/package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "coherent-gameface-checkbox",
-  "version": "3.1.0",
+  "version": "3.2.0",
   "description": "A checkbox component for Coherent Labs' Gameface.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config webpack-config.js --env prod",
+    "start": "webpack serve --config webpack-config.js --env dev --watch"
```

Comment:
Why do we need those scripts? The other package.json files are not having such scripts.

Reply:
We don't need them, I tested the webpack config directly in the component before adding it to the CLI and I've forgotten to remove the scripts.